### ISSUE TITLE
EpochLambda class for easily plugging in metrics

### ISSUE THREAD

**Issue by ntoxeg** [pytorch/ignite#358]
Basically something like `ROC_AUC` in contrib but such that you just plug a function that takes predictions and labels, returning a metric.

I wrote that to myself to reduce boilerplate so if you think it makes sense to include something like that upstream, then I'll add tests for it and submit a PR.

**Comment by vfdev-5** [comment]
@ntoxeg could you please detail how this would be different of `EpochMetric` . if you could provide some code here, would be helpful.

**Comment by ntoxeg** [comment]
After thinking more about this, it is perhaps true that this would not be different enough from `EpochMetric`. `ROC_AUC` is a very simple derived class from `EpochMetric` too though. Would it be better if I just wrote some explicit wrappers for metrics from Scikit-learn that I'd like included?

**Comment by vfdev-5** [comment]
@ntoxeg what kind of wrappings you are thinking ?

**Comment by vfdev-5** [comment]
@ntoxeg I think we can close this issue.
